### PR TITLE
fix(DB/Quest): Spanish text for Mebok Mizzyrix greeting

### LINKED ISSUE #16312: (Locale): Missing Spanish text for Mebok Mizzyrix.
### Current Behaviour

[Mebok Mizzyrix](https://wowgaming.altervista.org/aowow/?npc=3446) has no [Spanish localized text](https://github.com/azerothcore/azerothcore-wotlk/assets/38561765/3836e9d9-8117-4f02-ad18-6c88b0a4f43b).

### Expected Blizzlike Behaviour

He should have [localized text](https://github.com/azerothcore/azerothcore-wotlk/assets/38561765/9a762d63-b4bf-4e8b-9fa3-3428aa0ab12a).

### Source

Wrath Classic

### Steps to reproduce the problem

1. Launch the game with esES/esMX locale.
2. `.go c id 3446`
3. `.levelup 79`
4. Interact with him.
5. Observe his text.

### Extra Notes

Originally reported at https://github.com/chromiecraft/chromiecraft/issues/4433.

### AC rev. hash/commit

42a1650d3146e025d996ce5b9b5dfbd9bd8fbe88

### Operating system

Windows 10 x64

### Custom changes or Modules

[mod-ptr-template](https://github.com/heyitsbench/mod-ptr-template)

### PR DESCRIPTION
**Pull request by trdillon**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes #16312 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Valdifer** [comment]
Little thing:
It says 'el humana' when should be 'la humana'

**Review by Valdifer — CHANGES_REQUESTED** [review]
Little thing:
It says 'el humana' when should be 'la humana'

INSERT INTO `acore_world`.`quest_greeting_locale` (`ID`, `locale`, `Greeting`) VALUES (3446, 'esMX', '¡Sí, sí, sí! ¡Justo $gel:la $r que estaba buscando!$B$B¡Siéntate! Tenemos mucho que hablar.');
INSERT INTO `acore_world`.`quest_greeting_locale` (`ID`, `locale`, `Greeting`) VALUES (3446, 'esES', '¡Sí, sí, sí! ¡Justo $gel:la $r que estaba buscando!$B$B¡Siéntate! Tenemos mucho que hablar.');

**Comment by trdillon** [comment]
Thanks for pointing that out, I've added those changes.

**Comment by elthehablo** [comment]
What is the source on the text?

**Comment by trdillon** [comment]
I used the screenshot in the original issue under expected behavior.

**Comment by elthehablo** [comment]
Oh I missed that. My bad. Thanks for the PR

**Comment by elthehablo** [comment]
Need some changes on sql formatting though. Take a look at the [AC sql standards](https://www.azerothcore.org/wiki/sql-standards) please

**Comment by trdillon** [comment]
My apologies, I missed that part when I was checking out the wiki. The latest commit should conform to the standards.

**Comment by elthehablo** [comment]
It doesn't, unfortunately. Missing new line and delete before insert

**Comment by elthehablo** [comment]
Why did you close this?

File: data/sql/updates/pending_db_world/rev_1698965454224484600.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- Mebok Mizzyrix
+INSERT INTO `quest_greeting_locale` (`ID`, `locale`, `Greeting`) VALUES
+(3446, 'esMX', '¡Sí, sí, sí! ¡Justo $gel:la; $r que estaba buscando!$B$B¡Siéntate! Tenemos mucho que hablar.'),
+(3446, 'esES', '¡Sí, sí, sí! ¡Justo $gel:la; $r que estaba buscando!$B$B¡Siéntate! Tenemos mucho que hablar.');
```
Comment (elthehablo) [inline data/sql/updates/pending_db_world/rev_1698965454224484600.sql:4]:
```suggestion
--
DELETE FROM `quest_greeting_locale` WHERE `ID` = 3446 AND `locale` IN ('esMX', 'esES');
INSERT INTO `quest_greeting_locale` (`ID`, `locale`, `Greeting`) VALUES
(3446, 'esMX', '¡Sí, sí, sí! ¡Justo $gel:la; $r que estaba buscando!$B$B¡Siéntate! Tenemos mucho que hablar.'),
(3446, 'esES', '¡Sí, sí, sí! ¡Justo $gel:la; $r que estaba buscando!$B$B¡Siéntate! Tenemos mucho que hablar.');

```

**Pull request closed without merging**